fairmq/shmem/Manager.cxx

```diff
@@ -94,84 +101,155 @@ void Manager::StartMonitor(const std::string& id)
     }
 }
 
-void Manager::Interrupt()
+pair<bipc::mapped_region*, uint64_t> Manager::CreateRegion(const size_t size, const int64_t userFlags, RegionCallback callback, const string& path /* = "" */, int flags /* = 0 */)
 {
-}
+    try {
+        uint64_t id = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'id' can be reduced.](https://app.codacy.com/manual/FairRootGroup/FairMQ/pullRequest?prid=5424357)